### PR TITLE
I119 integrate backend and frontend signup

### LINKED ISSUE #119: Integrate backend and frontend signup
## Basic Information
**[Give a brief summary of what the issue is about]**

## Other Information
**[Are there any additional information that we should know about?]**

### PR DESCRIPTION
**Pull request by jsun1590**
## Change Summary
**[Briefly summarise the changes that you made. Just high-level stuff]**

### Change Form
**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [x] The pull request title has an issue number
- [x] The change works by "Smoke testing" or quick testing
- [x] The change has tests
- [ ] The change has documentation

## Other Information
**[Is there anything in particular in the review that I should be aware of?]**

# Related issue

- Resolve #119

### CODE REVIEW

**Comment by CodeWithTheDoctor** [comment]
The form doesn't seem to be functional.

I tried submitting this on the vercel deployment:
image

And the following error gets logged:
image

**Comment by CodeWithTheDoctor** [comment]
Omg, I totally forgot about that! I'll have a look again hahaha

**Comment by jsun1590** [comment]
But I should prolly handle errors on time out to backend, good point

File: client/cypress/e2e/auth/signup.spec.cy.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+describe('Signup', () => {
+  beforeEach(() => {
+    cy.visit('signup');
+  });
+  it('can signup', () => {
+    cy.get('input[name="First Name"').type('John');
```
Comment (CodeWithTheDoctor) [inline client/cypress/e2e/auth/signup.spec.cy.js:6]: Missing closing brackets.... mmmmmmm (yoda voice)

File: server/api/settings.py [hunk 2]
```diff
@@ -145,9 +148,9 @@
     ),
     "PAGE_SIZE": 10,
     "NON_FIELD_ERRORS_KEY": "errors",
-    "DEFAULT_PERMISSION_CLASSES": (
-        "rest_framework.permissions.IsAuthenticated",
-    ),
+    # "DEFAULT_PERMISSION_CLASSES": (
```
Comment (CodeWithTheDoctor) [inline server/api/settings.py:151]: I reckon get rid of unnecessary comments. Unless you think its sustainable

**Review by CodeWithTheDoctor — CHANGES_REQUESTED** [review]
Small changes, but everything else looks good

**Review by CodeWithTheDoctor — APPROVED** [review]
Looks good! send it in

**Pull request merged**